cfgov/v1/models/base.py

```diff
@@ -145,20 +145,16 @@ def alphabetize_authors(self):
         return sorted(author_names, key=lambda x: x.name.split()[-1])
 
     def generate_view_more_url(self, request):
-        from ..forms import FilterableListForm
         activity_log = CFGOVPage.objects.get(slug='activity-log').specific
-        form = FilterableListForm(parent=activity_log, hostname=request.site.hostname)
-        available_tags = [tag[0] for name, tags in form.fields['topics'].choices for tag in tags]
         tags = []
         index = activity_log.form_id()
         for tag in self.tags.slugs():
-            if tag in available_tags:
-                tags.append('filter%s_topics=' % index + urllib.quote_plus(tag))
+            tags.append('filter%s_topics=' % index + urllib.quote_plus(tag))
         tags = '&'.join(tags)
         return get_protected_url({'request': request}, activity_log) + '?' + tags
```

Comment:
This code could be made a little bit safer (no manual string combine or use of `&`) and simpler by using `urllib.urlencode`:

```py
>>> from urllib import urlencode
>>> urlencode([('filter%s_topics' % index, tag) for tag in self.tags.slugs()])
'filter0_topics=a&filter0_topics=b&filter0_topics=c&filter0_topics=d'
```